test/transformer/transformer_maps_test.dart

```diff
@@ -19,11 +18,22 @@ part "transformer_test_core.dart";
 /// We pretend that our main rule, [ThingWithDateSerializatinoRule] is generated
 /// so we can't or don't want to just modify it, so instead we subclass and
 /// modify the data before calling it.
-class SpecialThingWithDateRule extends ThingWithDateSerializationRule {
+class SpecialThingWithDateRule extends CustomRule {
+
+  CustomRule generated;
```

Comment:
I also think we will want a parameter for annotating a class you don't control, e.g. @Serializable(class: DateTime, ruleName: MyDateAndTimeRule). There was a mention in the misc thread today of someone who did not want to modify his model sources in any way, including annotations. I think being able to have the annotations separately is probably adequate for that, but not sure.